flask_resty/view.py

```diff
@@ -107,8 +108,62 @@ def validate_request_id(self, data, expected_id):
         if id != expected_id:
             raise ApiError(409, {'code': 'invalid_id.mismatch'})
 
+    def update_spec(self, path, spec, many=False, tag=False, **kwargs):
```

Comment:
and since there is a different implementation for `ApiView` vs `GenericModelView` will the client have to decide which one to chose? Or the mixin will have an `isinstance` to differentiate?